### DISCUSSION TITLE
Create multiple podcasts on the same platform

### DISCUSSION THREAD

**Discussion by kaxisangregorio** [PodcastGenerator/PodcastGenerator#760] [Q&A]
Hello! I would like to create several podcasts on a single platform and for each one to have its own cover, name, description, RSS address... How can I do it?

**Comment by CaptainPackers** [comment]
> Hello! I would like to create several podcasts on a single platform and for each one to have its own cover, name, description, RSS address... How can I do it?

I would like to do the same, but from my review of the code it does not appear to be developed to handle more than one podcast.

The easiest way to do it, as written, would be to replicate the entire code base in a separate directory in your public html folder. Kind of a waste because I think the code base is like 200 MB.

The alternative would require a major rewrite of the code base implementing arrays in order for one code base to handle multiple podcasts. I'm not going to do that.

It looks to me as though the developers made this change and commercialized it on rss.com, but have not, unfortunately released the implementation to the open source community.

-- Captain
